package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "server": "json-server --watch src/data/pets.json",
+    "server": "json-server --watch src/data/pets.json --port 2999",
```

Comment:
I don't love changing the port this runs on since it will run on port 3000 in Heroku.  For the solution I'd rather change the port that the node server runs on.  (It looks like you're making calls to `localhost:2999`.)

Reply:
Ok, I can change the node server to run on 2999 and let json server run on 3K